weather_sp/splitter_pipeline/file_name_utils.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+import typing as t
+
+_WEATHER_FILE_ENDINGS = ('.nc', '.cd', '.grib', '.grb', '.grb2', '.grib2')
```

Comment:
What about gzipped grib files? e.g. we see files with the extension `bz2` (and no other extension). Is there a way to process this without including a finite list of extensions?

Reply:
Splitter doesn't currently support zipped or tarred files, I think that's a separate feature request and would require more thought. I think to support that we'd likely unzip first anyway, and I don't know if we should really support rezipping the split files, would we then have to support every posible zip format?

Reply:
As far as opening files, I just did an experiment and it looks like Apache Beam filesystems does a good job of decompressing blobs in a wide range of formats. We just need to support the URI parsing so we can accept the files. 
A general principle that the weather tools have attempted to follow so far is to make an educated guess for processing, but ultimately give users control over processing. If the splitter had some way to override the allowed extensions, then it could be used in more cases.